### PR TITLE
Add 'Do Not Track' Support (use Plugin of outfox)

### PR DESCRIPTION
**Pull request by Nols1000**
Add 'Do Not Track' Support as described in this [article](http://www.outfox.com/do-not-track-for-google-analytics/).
Please consider adding this for a free Internet.

### CODE REVIEW

File: ganalytics.php [hunk 1]
```diff
@@ -82,16 +82,25 @@ private function getTrackingCode($scriptName, $objectName, $async=false)
...
           'trace-debug' =>  "window.ga_debug = {trace: true};",
           'create'      => "{$objectName}('create', '{$trackingId}', {$cookie_config});",
+		  'dnt'			=> "{$objectName}('require', 'dnt', {$dntOptions})",
           'anonymize'   => "{$objectName}('set', 'anonymizeIp', true);",
           'force-ssl'   => "{$objectName}('set', 'forceSSL', true);",
           'send'        => "{$objectName}('send', 'pageview');"
         ];
 
         if (!$this->config->get('plugins.ganalytics.debugTrace', false)) unset ($settings['trace-debug']);
+		if (!$this->config->get('plugins.ganalytics.dnt', false)) unset ($settings['dnt']);
```
Comment (escopecz) [inline ganalytics.php]: Could you please fix the indentation of the several rows that are off?

File: languages/de.yaml [hunk 2]
```diff
@@ -15,6 +15,13 @@ PLUGIN_GANALYTICS:
   POSITION_HELP: "Position des Codes im HTML-Dokument (Head oder Body)"
   POSITION_HEAD: "Kopf"
   POSITION_BODY: "Inhaltsende"
+  DO_NOT_TRACK_SECTION: "Do Not Track"
+  DO_NOT_TRACK: "Aktiviere 'Do Not Track'"
+  DO_NOT_TRACK_HELP: "Erlaubt Klienten, durch das senden des 'Do Not Track'-Headers, vom Tracking ausgeschlossen zu werden."
```
Comment (cworreschk) [inline languages/de.yaml]: Please use this translation `Erlaubt Nutzern, durch das Senden des 'Do Not Track'-Headers, vom Tracking ausgeschlossen zu werden.`

File: languages/de.yaml [hunk 3]
```diff
@@ -15,6 +15,13 @@ PLUGIN_GANALYTICS:
...
+  DO_NOT_TRACK_LOG: "Aktiviere Protokollierungsmodus"
+  DO_NOT_TRACK_LOG_HELP: "Beachten Sie, dass, während Sie das Plugin im Protokollierungsmodus verwenden, Daten an Google Analytics gesendet werden, auch wenn 'Do Not Track' aktiviert ist. Erst wenn Sie den Protokollierungsmodus deaktivieren, werden keine Daten an Google Analytics gesendet. Sie sollten nur den Protokollierungsmodus verwenden, um herauszufinden, welcher Anteil von Besuchen / Benutzern Sie verlieren, nachdem Sie die Protokollierung deaktiviert und die 'Do Not Track'-Unterstützung aktiviert haben."
+  DO_NOT_TRACK_DIMENSION: "Google Analytics Dimension"
+  DO_NOT_TRACK_DIMENSION_HELP: "Google Analytics Dimension to log the 'Do Not Track'-Status (https://goo.gl/i15S6B)"
```
Comment (cworreschk) [inline languages/de.yaml]: Please use this translation `Google Analytics Dimension um den 'Do Not Track'-Status zu protokollieren (https://goo.gl/i15S6B)`

**Comment by Nols1000** [comment]
Fixed everything you mentioned.

**Comment by cworreschk** [comment]
I'm currently testing your code. If it's successful I'lll merge your pull request.

**Comment by cworreschk** [comment]
I'm sorry, but it seems that your code doesn't work properly.

GA says `No plugin url set for "dnt".`

That's because you've included the `//storage.googleapis.com/outfox/dnt_min.js` before the global GA object was instantiated.

**Comment by escopecz** [comment]
@Nols1000 any update or can I close this PR?

**Comment by Nols1000** [comment]
No update.

**Pull request closed without merging**